code/game/objects/structures/crates_lockers/closets/utility_closets.dm

```diff
@@ -30,51 +30,39 @@
 		if ("small")
 			new /obj/item/weapon/tank/emergency/oxygen(src)
 			new /obj/item/weapon/tank/emergency/oxygen(src)
+			new /obj/item/weapon/rig/emergency(src)
```

Comment:
why giving out a rig in every option, that seems far too much

Reply:
It is a shitty, emergency rig that only has the purpose of keeping the wearer from suffocating. It replaces the emergency helmet and suit and has the same stats as it's counterpart, the standard emergency space suit.